recipes/botcity-maestro-sdk/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "botcity-maestro-sdk" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e02887fa92721ee488e4458669c0774f941e3928a8f4193c88972eb2adb4a1ee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - requests
```

Comment:
Can you please drop `requests` from the `host` requirements? I'm pretty sure it isn't needed at build time.

Reply:
@mwcraig thanks for the review. I addressed your comment via https://github.com/conda-forge/staged-recipes/pull/15265/commits/347d9ecd357c386d74802475f7237b0cc6d313a0